### ISSUE TITLE
Explore Nextcloud login and additional integrations

### ISSUE THREAD

**Issue by icidasset** [icidasset/diffuse#233]
I can't determine if it's possible to use the Nextcloud login system from reading its documentation. That said, I feel like it should be possible, so we should explore that option: https://docs.nextcloud.com/server/15/developer_manual/client_apis/LoginFlow/index.html I'm not a Nextcloud user myself, so any help would be appreciated 🙏 

---

We've also had several comments that the integration with
Nextcloud could be better, but I don't know how to improve that.

> I think the first useful thing would be to have a link in the Nextcloud menu, so we can just access Diffuse.

**Comment by reesericci** [comment]
what is the app for?

**Comment by reesericci** [comment]
the nextcloud app

**Comment by icidasset** [comment]
Not sure exactly. Someone asked me to submit it to the Nextcloud appstore or something like that, and if I recall correctly, people can install the app if it's on the store? 🤷 And this login thing would enable people to store the application data on their Nextcloud, but not entirely sure it's possible.

**Comment by reesericci** [comment]
I can install it but it seems to do nothing. what is it supposed to do?

**Comment by reesericci** [comment]
it just makes no sense

**Comment by reesericci** [comment]
I get a webdav error when using nextcloud webdav: ```Method PROPFIND not allowed: read only access ```

**Comment by icidasset** [comment]
Hmm ok, thanks! I'll take a look when I have some time.

**Comment by pianshanchou** [comment]
It looks great. I installed it in nextcloud. Now I have a question. How do I turn it on?

**Comment by icidasset** [comment]
Thanks for the responses everyone! I removed the app from the store.

I only added it because someone asked me, but I never tested it.
I tried to figure out how to make it work, but the Nextcloud documentation doesn't tell me anything.
It doesn't really make sense, so I removed it from the store.

I designed the app to be used mainly in the browser:
https://diffuse.sh
https://nightly.diffuse.sh

WebDAV should work, I've tested it with https://github.com/hacdias/webdav
I did find an issue with it though, which will be fixed in v3.
That said, I have no clue how to properly configure WebDAV with Nextcloud 🤷‍♂️
